crates/holochain_p2p/src/types/actor.rs

```diff
@@ -18,45 +18,45 @@ pub struct GetValidationPackage {
 
 #[derive(Clone, Debug)]
 /// Get options help control how the get is processed at various levels.
-/// Fields tagged with `[Network]` are network-level controls.
-/// Fields tagged with `[Remote]` are controls that will be forwarded to the
+/// Fields tagged with ```[Network]``` are network-level controls.
+/// Fields tagged with ```[Remote]``` are controls that will be forwarded to the
 /// remote agent processing this `Get` request.
 pub struct GetOptions {
-    /// [Network]
+    /// ```[Network]```
```

Comment:
What do triple ticks mean?

Reply:
When I use single ticks with the brackets, rustdoc interprets it as an intra-doc link. With triple ticks it'll be printed as `code` and with brackets, but not seen as intra-doc link.